NameLayerMain/src/vg/civcraft/mc/namelayer/group/Group.java

```diff
@@ -317,23 +318,38 @@ public PlayerType getCurrentRank(UUID uuid) {
 	 * @param type- The PlayerType to add. If a preexisting PlayerType is found, 
 	 * it will be overwritten.
 	 */
-	public void addMember(UUID uuid, PlayerType type) {
+	
+	public void addMember(UUID uuid, PlayerType type){
+		addMember(uuid,type,true);
```

Comment:
Consistency check QQ: here the prior interface will trigger asynchronous insertion instead of prior, synchronous behavior.

I haven't surveyed within this pull who uses `addMember(uuid, type)` but are you confident the consumers are all internal and won't mind? Also I'm concerned there might be a hidden double-db-save and double-mercury-message in here somewhere .... some combo between GroupManager and these calls here on Group. 

Feel free to tell me I'm wrong, haven't traced it fully yet.